### DISCUSSION TITLE
Fully Defined Sketch Color Change

### DISCUSSION THREAD

**Discussion by TheMakerTales** [hlorus/CAD_Sketcher#61] [Feature Requests]
To go with the checkmark of a fully defined sketch. It would be nice if it would support a line colour change or a constraint colour change.

Eg.
Under defined sketches = Default color
Over defined sketches = Red ( This seems to already be implemented for constraints)
Fully defined sketcher = Green or Blue

**Comment by Intuos5** [comment]
I think it would be best to implement the colours used in Fusion 360, so Blue for unconstrained, black for fully constrained. The blue colour stands out and indicates there is work to be done. Black indicates regular lines and does not need to stand out, fully defined means it is ready to be worked on with modifiers. Purple is used for projection.

**Comment by hlorus** [comment]
Also see #233
